plugins/rbac/src/components/ConditionalAccess/AddNestedConditionButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Box } from '@material-ui/core';
+import Tooltip from '@material-ui/core/Tooltip';
+import HelpOutlineIcon from '@material-ui/icons/HelpOutline';
+
+export const AddNestedConditionButton = () => {
+  const tooltipTitle = () => (
+    <div>
+      <p style={{ textAlign: 'center' }}>
+        Nested conditions are <b>1 layer rules within a main condition</b>. It
+        lets you allow appropriate access by using detailed permissions based on
+        various conditions. You can add multiple nested conditions.
```

Comment:
IMO nested condition definition can be improved. We should explain what a nested condition is and then add that the UI currently supports only 1 level of nesting i.e multiple criteria based conditions can only be added to the upper most criteria.

@AndrienkoAleksandr @parvathyvr any suggestions on the definition?

Reply:
> IMO nested condition definition can be improved. We should explain what a nested condition is and then add that the UI currently supports only 1 level of nesting i.e multiple criteria based conditions can only be added to the upper most criteria.
> 
> @AndrienkoAleksandr @parvathyvr any suggestions on the definition?

I find this text fairly clear in explaining what a nested condition is and discussing your suggestion. It mentions the limitation of having only 1 layer and how it has been incorporated into the main condition. I'm unsure about what you feel is still unclear or missing. Could you please explain what is missing/unclear?

Reply:
`Nested conditions are <b>1 layer rules within a main condition`
@ShiranHi I find above line confusing as nested conditions can have multiple levels. It looks like we are stating that in general nested conditions can have only 1 level. But its just a suggestion, if you think it looks correct then I am fine with it :)

Reply:
cc: @invincibleJai

Reply:
> I find above line confusing as nested conditions can have multiple levels. It looks like we are stating that in general nested conditions can have only 1 level whereas its the limitation of the UI we are providing.

Got it, I see your point. I think that if we allow for one level of nested conditions on the UI, this is the story we should communicate to users, unless it will be confusing. I've tried some other wording but it feels that this sentence is the best option for now. Unless you have a better suggestion, let's stick with this version, as it was suggested by our UX copywriter.